src/boosting/cuda/cuda_score_updater.cu

```diff
@@ -22,7 +22,7 @@ __global__ void AddScoreConstantKernel(
 
 void CUDAScoreUpdater::LaunchAddScoreConstantKernel(const double val, const size_t offset) {
   const int num_blocks = (num_data_ + num_threads_per_block_) / num_threads_per_block_;
-  Log::Warning("adding init score = %f", val);
+  Log::Debug("Adding init score = %lf", val);
```

Comment:
@StrikerRUS Thanks for this! Did not noticed that this debug information still exists in the code base.